packages/editor/src/components/parts/web-service/WebServicePart.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { DeepPartial, render, renderHook, screen } from 'test-utils';
+import { ElementData, InscriptionValidation, WebserviceStartData } from '@axonivy/inscription-protocol';
+import { PartStateFlag } from '../../editors';
+import { useWebServicePart } from './WebServicePart';
+
+const Part = () => {
+  const part = useWebServicePart();
+  return <>{part.content}</>;
+};
+
+describe('WebServicePart', () => {
+  function renderPart(data?: WebserviceStartData) {
+    render(<Part />, {
+      wrapperProps: { data: data && { config: data } }
+    });
+  }
+
+  test('empty data', async () => {
+    renderPart();
+    expect(screen.queryByText('Permission')).toBeInTheDocument();
+    expect(screen.queryByText('Exception')).toBeInTheDocument();
+  });
+
+  test('full data', async () => {
+    renderPart({
+      permission: {
+        anonymous: false,
+        error: '>> Ignore Exception',
+        role: 'Test'
+      },
+      exception: {
+        condition: '0===0',
+        enabled: true,
+        message: 'hallo'
+      }
+    });
+    expect(screen.queryByText('Permission')).toBeInTheDocument();
+    expect(screen.queryByText('Exception')).toBeInTheDocument();
+  });

```

Comment:
more or less same test... As you test the data already in the other tests, I think it is fine to only test the empty state here...